source/Plugins.cpp

```diff
@@ -96,6 +108,9 @@ const Plugin *Plugins::Load(const string &path)
 
 	plugin->path = path;
 
+	plugin->aboutText = Files::Read(path + "about.txt");
```

Comment:
The about text is already read above, why read it again?

Reply:
```suggestion
```